### PR TITLE
Add letters to English QWERTZ and QWERTY and add Czech xml

### PR DESCRIPTION
**Pull request by ikanakova**

### CODE REVIEW

**Comment by ikanakova** [comment]
It's clear to me that I have to make more changes somewhere, but I didn't dare - I'm not that good a programmer :-(

File: app/src/main/res/xml/keys_letters_english_qwerty.xml [hunk 1]
```diff
@@ -68,17 +68,17 @@
...
         <Key
             app:keyLabel="i"
-            app:popupCharacters="íìî8ïī"
+            app:popupCharacters="íìî8ïīį"
             app:popupKeyboard="@xml/keyboard_popup_template"
             app:topSmallNumber="8" />
         <Key
             app:keyLabel="o"
-            app:popupCharacters="őöøóôòõ9ō"
+            app:popupCharacters="őöøóôòõ9ōœ"
```
[the remark this answers is not in the document]
Comment (ikanakova) [reply app/src/main/res/xml/keys_letters_english_qwerty.xml:81]: > why would this be needed in an english keyboard?

I think it's useful if someone wants to use classic English QWERTY and not a local keyboard mutation, but needs some of these characters.

[the remark this answers is not in the document]
Comment (tibbi) [reply app/src/main/res/xml/keys_letters_english_qwerty.xml:81]: I dont really want to add such rare characters there that noone asked for. If the PR is about the czech language, avoid changing unrelated things please

**Comment by tibbi** [comment]
and the new file isnt even added anywhere in the code, how have you tested if it works well?

**Comment by tibbi** [comment]
that file is almost the same as the english one, not worth adding.. Thanks though

**Pull request closed without merging**